### PR TITLE
Fixed deprecated BaseBlockService usage

### PR DESCRIPTION
**Pull request by vladfilimon**
sonata-project/block-bundle, on 3.x branch, BaseBlockService @deprecated since 3.2, to be removed with 4.0

## Subject

Fixed deprecated `BaseBlockService` usage, in the 3.x branch of sonata-project/block-bundle

### CODE REVIEW

File: Block/AuditBlockService.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 use SimpleThings\EntityAudit\AuditReader;
 use Sonata\AdminBundle\Form\FormMapper;
-use Sonata\BlockBundle\Block\BaseBlockService;
+use Sonata\BlockBundle\Block\Block\Service\AbstractBlockService;
```
Comment (OskarStark) [inline Block/AuditBlockService.php]: Block\Block ?

**Comment by OskarStark** [comment]
Do we need to increase the dependency of block-bundle?

**Comment by vladfilimon** [comment]
The block-bundle does not need to be increased. 
The dependency comes from "sonata-project/admin-bundle": "^3.1"
And, in admin-bundle composer.json: "sonata-project/block-bundle": "^3.2"

The deprecation occured on the 3.x branch of block-bundle.

**Comment by vladfilimon** [comment]
I cannot understand why StyleCI fails

**Comment by greg0ire** [comment]
> I cannot understand why StyleCI fails

Click the details link ;)

**Comment by OskarStark** [comment]
> I cannot understand why StyleCI fails

the order of the use statements is wrong @vladfilimon

**Comment by greg0ire** [comment]
Anyone who want to merge this, dismiss my review and squash-merge the commits. Alternatively @vladfilimon , you can squash the commits yourself and provide a nice commit message.

**Comment by greg0ire** [comment]
Wait, why was this on master?

**Comment by OskarStark** [comment]
> Wait, why was this on master?

Should be merged on the other branch too, IMO

@greg0ire can you cherry pick?

**Comment by greg0ire** [comment]
Sure, see #648

**Pull request merged**